guides/source/active_job_basics.md

```diff
@@ -388,6 +389,23 @@ class GuestsCleanupJob < ApplicationJob
 end
 ```
 
+A callback that `throws :abort` will [halt the callback chain](https://api.rubyonrails.org/classes/ActiveSupport/Callbacks.html)
+and skip the execution of any subsquent before, around and after callbacks:
+
+```ruby
+class GuestsCleanupJob < ApplicationJob
+  queue_as :default
+
+  before_enqueue do
+    throw :abort if ENV.fetch("DISABLE_GUESTS_CLEANUP_JOB", false)
+  end
+
+  def perform
+    # Do something later
+  end
+end
+```
+
```

Comment:
The opening paragraph of the "Callbacks" section mentions "macro-style class method" (line 371), and line 409 expands on this some more. Adding this section here somewhat breaks the flow.
So I think this section should be moved, probably after the "Available Callbacks" section where we first mention `before_enqueue`.
It should also probably get a header, something like`### Halting Callbacks`.